amethyst_animation/src/skinning/systems.rs

```diff
@@ -16,16 +14,12 @@ use super::resources::*;
 /// System for performing vertex skinning.
 ///
 /// Needs to run after global transforms have been updated for the current frame.
-#[derive(Debug, SystemDesc)]
-#[system_desc(name(VertexSkinningSystemDesc))]
+#[derive(Debug)]
 pub struct VertexSkinningSystem {
```

Comment:
This needs migration to legion as well, right? (The system by itself)

Reply:
Migrated.